src/main/java/com/programmers/vouchermanagement/controller/api/VoucherApiController.java

```diff
@@ -0,0 +1,50 @@
+package com.programmers.vouchermanagement.controller.api;
+
+import com.programmers.vouchermanagement.dto.voucher.request.CreateVoucherRequestDto;
+import com.programmers.vouchermanagement.dto.voucher.request.GetVouchersRequestDto;
+import com.programmers.vouchermanagement.dto.voucher.request.UpdateVoucherRequestDto;
+import com.programmers.vouchermanagement.dto.voucher.response.VoucherResponseDto;
+import com.programmers.vouchermanagement.service.VoucherService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/vouchers")
```

Comment:
api인지와 버전까지 명시해주면 좋습니다.
```suggestion
@RequestMapping("api/v1/vouchers")
```